client/extensions/woocommerce/app/order/order-customer/index.js

```diff
@@ -4,26 +4,79 @@
  */
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
 import AddressView from 'woocommerce/components/address-view';
+import Button from 'components/button';
 import Card from 'components/card';
+import CustomerAddressDialog from './dialog';
+import { editOrder } from 'woocommerce/state/ui/orders/actions';
+import { isCurrentlyEditingOrder, getOrderWithEdits } from 'woocommerce/state/ui/orders/selectors';
 import getAddressViewFormat from 'woocommerce/lib/get-address-view-format';
+import { getOrder } from 'woocommerce/state/sites/orders/selectors';
+import { getSelectedSiteWithFallback } from 'woocommerce/state/sites/selectors';
 import SectionHeader from 'components/section-header';
 
 class OrderCustomerInfo extends Component {
 	static propTypes = {
+		orderId: PropTypes.number.isRequired,
```

Comment:
I know, I know, me and my proptype comments 😆 `isEditing`, `site`, and `siteId` are now being passed in the `connect` - though looks like `site` isn't being used, only the id.